mix.exs

```diff
@@ -49,11 +49,15 @@ defmodule SignsUi.Mixfile do
       {:phoenix_live_reload, "~> 1.0", only: :dev},
       {:gettext, "~> 0.11"},
       {:cowboy, "~> 1.0"},
+      {:plug_cowboy, "~> 1.0"},
       {:poison, "~> 3.1"},
       {:httpoison, "~> 1.0"},
       {:ex_aws, "~> 2.0"},
       {:ex_aws_s3, "~> 2.0"},
-      {:timex, "~> 3.1"}
+      {:timex, "~> 3.1"},
+      {:guardian, "~> 1.0"},
+      {:ueberauth, "~> 0.1"},
+      {:ueberauth_cognito, git: "https://github.com/mbta/ueberauth_cognito.git"}
```

Comment:
You should release this at some point. Another notch in our open-source belt!

Reply:
Yep, we plan to! We have a ticket in our backlog for that.